src/tests/test_magic.py

```diff
@@ -195,6 +195,162 @@ def test_persist_bare(ip):
     assert result.error_in_exec
 
 
+def query_and_save_as_dataframe(ip, table_name, limit):
+    save_df_name = f"df_{table_name}"
+    ip.run_cell(f"results = %sql SELECT * FROM {table_name} LIMIT {limit};")
+    ip.run_cell(f"{save_df_name} = results.DataFrame()")
+    return save_df_name
+
+
+@pytest.mark.parametrize(
+    "test_table, expected_result",
+    [
+        ("test", [(0, 1, "foo")]),
+        ("author", [(0, "William", "Shakespeare", 1616)]),
+        (
+            "website",
+            [
+                (
+                    0,
+                    "Bertold Brecht",
+                    "https://en.wikipedia.org/wiki/Bertolt_Brecht",
+                    1954,
+                )
+            ],
+        ),
+        ("number_table", [(0, 4, -2)]),
+    ],
+)
+def test_persist_replace_abbr_no_override(ip, test_table, expected_result):
+    # New saved dataframe length -> 1
```

Comment:
can we remove this comment? Not like before, I'm not sure the length is something we should specifically mention here.